tests/adapter/dbt/tests/adapter/basic/test_timestamps.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from dbt.tests.util import check_relation_has_expected_schema, run_dbt
```

Comment:
@jtcohen6 migrate tests like this?

Reply:
yup!

by convention, we have reserved `adapter/basic` as, the basic tests to start with when developing/testing a new adapter: https://docs.getdbt.com/docs/contributing/adapter-development/4-testing-a-new-adapter#getting-started-running-basic-tests

I think this specific test might want to live in [`dbt.tests.adapter.utils.data_types`](https://github.com/dbt-labs/dbt-core/tree/main/tests/adapter/dbt/tests/adapter/utils/data_types)

Reply:
I thought those were for data type casting and I didn't want it to get confused with the existing type_timestamp tests

Reply:
ah you're right! maybe just a new subfolder under [`utils`](https://github.com/dbt-labs/dbt-core/tree/main/tests/adapter/dbt/tests/adapter/utils)

Reply:
moved to `utils`